model/lesson.js

```diff
@@ -122,6 +139,40 @@ async function getLessonById(data) {
     );
 }
 
+/*Fetches verses in a lesson based*/
+//ASSUMPTIONS: Verses in a Lesson are contigous
+async function getLessonVerses(data) {
+    console.log(data);
+    var invalid = utils.simpleValidation(data, {
```

Comment:
ur already validating inside of getlessonbyid, and ur already checking the return value.